WordPress/gutenberg
# [RNMobile] Introduce grouping in the block settings inspector

## Description
Ref to gb-mobile https://github.com/wordpress-mobile/gutenberg-mobile/pull/1397
* added section title in block settings 
* added new separator type called `topFullWidth` for setting buttons
* added new component `PanelActions` for handling action buttons in block settings. It accepts a prop called `actions` which is an array of button objects with properties `label`, `onPress`, `labelStyle`, e.g:
```
const actions = [ { label: 'Clear All Settings', onPress: this.onClearSettings, labelStyle: styles.clearSettingsButton } ];
```


## How has this been tested?
Tested on both platforms.

1. Add an image.
2. Press and icon <img width="30" src="https://user-images.githubusercontent.com/22746080/66471147-23979080-ea8b-11e9-9d12-61350c9ae97d.png" />
3. Check the `BlockSettings` inspector

## Screenshots 
android | ios
--- | ---
![Screenshot 2019-10-02 at 11 39 07](https://user-images.githubusercontent.com/22746080/66034380-6dbbc780-e509-11e9-978a-af6c89f7b159.png) | ![Screenshot 2019-10-02 at 11 24 44](https://user-images.githubusercontent.com/22746080/66034344-5f6dab80-e509-11e9-9730-c13010d06ce2.png)


## Checklist:
